tests/arch/arc/arc_dsp_sharing/src/test_common.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2022 Synopsys
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define MAX_TESTS	500
+
+/*
+ * Test Thread Parameters
+ */
+#define THREAD_STACK_SIZE	(1024 + CONFIG_TEST_EXTRA_STACK_SIZE)
+
+#define THREAD_HIGH_PRIORITY	5
+#define THREAD_LOW_PRIORITY	10
+
+#if defined(CONFIG_ISA_ARCV2)
+#define THREAD_DSP_FLAGS		(K_DSP_REGS | K_AGU_REGS)
+#else
+#define THREAD_DSP_FLAGS		(K_DSP_REGS)
```

Comment:
Is it worth to have this 'else' branch? We support this test for ARCv2 only, as I can see from `tests/arch/arc/arc_dsp_sharing/testcase.yaml`